app/src/main/java/io/github/polysmee/appointments/AppointmentActivity.java

```diff
@@ -23,7 +23,7 @@
 import io.github.polysmee.R;
 import io.github.polysmee.appointments.fragments.AppointmentCreationAddUserFragment;
 import io.github.polysmee.appointments.fragments.AppointmentCreationBanUserFragment;
-import io.github.polysmee.interfaces.DataPasser;
+import io.github.polysmee.database.DataPasser;
```

Comment:
Why DataPasser is in the database package ?

Reply:
having a package named interface makes no sense, so i dragged all three into database a week ago, but since nobody uses it it apparently passed compilation when it should not have. Where do I put it ?